pkg/bunapp/app.go

```diff
@@ -56,6 +57,12 @@ func New(configPath string, opts ...fx.Option) (*fx.App, error) {
 		fx.Provide(initGRPC),
 		fx.Provide(fx.Annotate(initTaskq, fx.As(new(taskq.Queue)))),
 		fx.Provide(newHTTPClient),
+		fx.WithLogger(func() fxevent.Logger {
+			logger := slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{
```

Comment:
Let's use the existing slog logger.

And we probably just need to make the defaul log leve slog.LevelWarn.